proto/process/agent.proto

```diff
@@ -603,3 +606,21 @@ message ContainerStatus {
 	string state = 5;
 	string message = 6;
 }
+
+message ConnectionsTelemetry {
+	int64 kprobesTriggered = 1;
+	int64 kprobesMissed = 2;
+	int64 conntrackRegisters = 3;
+	int64 conntrackRegistersDropped = 4;
+	int64 dnsPacketsProcessed = 5;
+	int64 connsOpened = 6;
+}
```

Comment:
These two structs look identical. Let's prefix each field (or at least the type name) with `Monotonic`  to indicate that these counters will go up monotonically. We use this naming convention elsewhere in the system-probe and it helps a lot with readability.